docs/docs/troubleshooting.md

```diff
@@ -110,3 +110,7 @@ For the `nRF52840`, the value `PLUS_8` can be set to any multiple of four betwee
 ### Other notes and warnings
 
 - If you want to test bluetooth output on your keyboard and are powering it through the USB connection rather than a battery, you will be able to pair with a host device but may not see keystrokes sent. In this case you need to use the [output selection behavior](../docs/behaviors/outputs.md) to prefer sending keystrokes over bluetooth rather than USB. This might be necessary even if you are not powering from a device capable of receiving USB inputs, such as a USB charger.
```

Comment:
Why remove this?

Reply:
There is no removal. That floating `-` is a single bullet point from the existing section and it is still there.

Reply:
Oh! Rendering funkiness in the mobile app. Thanks.